File: src/data/courses/dsa/two-pointers-sliding-window/two-pointers-opposite-direction/container-most-water/code/python/container_most_water_sol1.py
class Solution:
    def maxArea(self, height):
        max_area = 0
        for i in range(len(height)):
            for j in range(i+1, len(height)):
                max_area = max(max_area, min(height[i],height[j])*(j-i))
        return max_area

File: src/data/courses/dsa/two-pointers-sliding-window/two-pointers-opposite-direction/container-most-water/code/cpp/container_most_water_sol1.cpp
class Solution {
public:
    // Brute Force: O(n²) — Check all pairs
    int maxArea(vector<int>& height) {
        int maxA = 0;
        for (int i = 0; i < height.size(); i++)
            for (int j = i+1; j < height.size(); j++)
                maxA = max(maxA, min(height[i], height[j]) * (j - i));
        return maxA;
    }
};